bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
potrebujeme tenhle soubor? Ty pouzivas bin/console? Osobne jsem to nikdy nepouzil